bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImplTest.java

```diff
@@ -230,6 +231,31 @@ void testGetThankYouMessage() throws Exception {
         assertEquals("Thank you for submitting.", formContainer.getThankYouMessage());
     }
 
+    @Test
+    void testGetIsEdgeDeliveryRequest() throws Exception {
```

Comment:
The tests seem incomplete for this feature.
we should also test if the request attribute has the property set, then the markup generated should contain pre-code tags.